site/hr_HR.all.json

```diff
@@ -135,16 +135,16 @@
     "other": "Udio"
   },
   "social_media_buttons_facebook": {
-    "other": "Podijeli na Facebooku"
+    "other": "Facebook"
   },
   "social_media_buttons_twitter": {
-    "other": "Podijelite na Twitteru"
+    "other": "Cvrkut"
```

Comment:
I can see Twitter being translated in a few of these files... that's not supposed to happen right?

Reply:
I looked at some old commits and it was translated in some files and not in others. I'll go through and change them to English.